docstr_coverage/cli.py

```diff
@@ -173,6 +174,8 @@ def parse_ignore_names_file(ignore_names_file: str) -> tuple:
 @click.option("--failunder", type=float, help="Deprecated. Use --fail-under")
 def execute(paths, **kwargs):
     """Measure docstring coverage for `PATHS`"""
+    if os.path.exists(".docstr.yaml"):
```

Comment:
I think we can remove this `if` check, since it's also performed on line 20 of `configFile.py`.

Just set ```paths, kwargs = readConfigFile(paths, kwargs)```